eng/Directory.Build.Common.props

```diff
@@ -135,7 +136,7 @@
     <SupportsNetStandard20 Condition="'$(SupportsNetStandard20)' == ''">false</SupportsNetStandard20>
 
     <RequiredTargetFrameworks>net452;netstandard1.4;net461;netstandard2.0</RequiredTargetFrameworks>
-    <RequiredTargetFrameworks Condition="'$(SupportsNetStandard20)' == 'true'">netstandard2.0</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks Condition="'$(SupportsNetStandard20)' == 'true'">net8.0;netstandard2.0</RequiredTargetFrameworks>
```

Comment:
Does this add a target for `net8.0` to all clients in the azure-sdk-for-net repo?  Has @KrzysztofCwalina given the 👍 on this?

Reply:
Yes, this was the approved plan that we talked over for target frameworks in the Core meeting around 9 months ago and was blessed by Krzysztof and Ted.   We just took a while to move on it.   Going forward, we'll be following the plan that we outlined there:

- Always `netstandard2.0`
- Current LTS
- Since months after an LTS hits end-of-life, we remove the explicit target and allow fallback.


see:  https://gist.github.com/jsquire/d7a98cb42b9eaa9e3b942a7acd33a233

Reply:
@jsquire it would be good to port that gist somewhere in the repo so we can find and refer back to it in the future as we try to understand the plan.

Reply:
We still haven't figured out a good place to document it that doesn't constitute a public guarantee.   It's more a default working plan than anything that we want to put out there as iron-clad.